### PR TITLE
Tarzan-mode

### PR DESCRIPTION
**Pull request by teoman002**
…c) yavl/teeworlds-infclassR#78

### CODE REVIEW

**Comment by teoman002** [comment]
@bretonium @yavl tested this on his server. We can merge it.

File: src/game/server/entities/character.cpp [hunk 1]
```diff
@@ -554,10 +554,7 @@ void CCharacter::UpdateTuningParam()
 	{
 		pTuningParams->m_Gravity = 0.0f;
 	}
-	if(GetPlayer()->HookProtectionEnabled())
-	{
 		pTuningParams->m_PlayerHooking = 0;
```
Comment (bretonium) [inline src/game/server/entities/character.cpp]: indentation should be fixed here

Comment (teoman002) [reply src/game/server/entities/character.cpp]: Thx xD

**Comment by teoman002** [comment]
closed #78

**Pull request merged**